src/components/NewTask.tsx

```diff
@@ -78,23 +79,56 @@ export const NewTask = ({ locales }: Props) => {
 
   const createTask = async () => {
     if (!context) {
-      console.error('Missing context')
+      toast.push({
+        title: 'Unable to create task: missing context',
+        status: 'error',
+        closable: true,
+      })
       return
     }
+
     setIsBusy(true)
+
     context
       .exportForTranslation(context.documentId)
-      .then(serialized => {
-        return context.adapter.createTask(
+      .then(serialized =>
+        context.adapter.createTask(
           context.documentId,
           serialized,
           selectedLocales as string[],
           context.secrets,
           selectedWorkflowUid
         )
-      })
+      )
       .then(() => {
+        toast.push({
+          title: 'Job successfully created',
+          status: 'success',
+          closable: true,
+        })
+
+        /** Reset form fields */
+        setSelectedLocales([])
+        setSelectedWorkflowUid('')
         setIsBusy(false)
+
+        /** Update task data in TranslationView */
+        refreshTask()
+      })
+      .catch(err => {
+        let errorMsg
+        if (err instanceof Error) {
+          errorMsg = err.message
+        } else {
+          errorMsg = err ? String(err) : null
+        }
+
+        toast.push({
+          title: `Error creating translation job`,
+          description: errorMsg,
+          status: 'error',
+          closable: true,
+        })
       })
```

Comment:
Just a comment, if the `catch` function is called before the `setIsBusy(false)` the `isBusy` will get stuck as `true`...
In this case you have two options, maybe more :smile::
- call `setIsBusy(false)` at the `catch` function too;
- remove the  `setIsBusy(false)` at the `then` function and add in the final of the promise chain a [function called `finally`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/finally) and add the  `setIsBusy(false)`